SupportClient/SupportClient/Views/Cells/CustomerServiceTableViewCell.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  CustomerServiceTableViewCell.swift
+//  SupportClient
+//
+//  Created by Yana Sang on 5/8/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+class CustomerServiceTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = "customerServiceTableViewCell"
+
```

Comment:
Is there a reason why everything is indented 2x?

Reply:
no lmfao didn't even realize